modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/index/meta/IndexMetaKey.java

```diff
@@ -38,4 +38,23 @@ public IndexMetaKey(int indexId) {
     public int indexId() {
         return indexId;
     }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
Is this just to make a test a bit more tidy? I'm not sure we should add it here. In the test an `ArgumentCaptor` can be used to verify that the correct key is used for destruction.

Reply:
You are correct, but I don't see anything harmful in these methods being here, they might be useful.